safe_transaction_service/history/models.py

```diff
@@ -1238,8 +1238,8 @@ class MultisigTransaction(TimeStampedModel):
     signatures = models.BinaryField(null=True, blank=True)  # When tx is executed
     nonce = Uint256Field(db_index=True)
     failed = models.BooleanField(null=True, blank=True, default=None, db_index=True)
-    origin = models.CharField(
-        null=True, blank=True, default=None, max_length=200
+    origin = models.JSONField(
+        null=False, default=dict
```

Comment:
Remove `null=False`, as it's the default behaviour

Reply:
Done [here](https://github.com/safe-global/safe-transaction-service/pull/1177/commits/36e2571f462ce4fda01edc71d867f1b69e3a7dc0)